src/vasp_driver.py

```diff
@@ -148,17 +148,27 @@ def continue_job(*argv, **kwargs):
 
             # Count the number of possible jobs
             
-            count_POSCAR = len(glob.glob("*.POSCAR")) 
+            count_POSCAR = len(glob.glob("*.in")) 
```

Comment:
I'm going to hold off on merging this PR because there seem to be edits (e.g., this line) arising from hacking in QE that would break functionality (e.g., for VASP).